src/test/kotlin/notification/application/NotificationGetterTest.kt

```diff
@@ -1,45 +1,60 @@
 package notification.application
 
+import notification.domain.Notification
 import notification.domain.NotificationRepository
 import notification.mothers.NotificationMother
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.mockito.Mockito
 import shared.domain.IdentifierGenerator
 import shared.mothers.IdentifierMother
+import java.util.*
```

Comment:
Why is this needed? Aren't we using the kotlin collections API?

Reply:
Probably an unused import that I forgot to remove, I'll check.